### PR TITLE
[RFR] Fix dev logs and standardize koa error

### PR DESCRIPTION
**Pull request by Kmaschta**

### CODE REVIEW

File: src/api/server.js [hunk 1]
```diff
@@ -120,20 +120,29 @@ app.use(koaMount('/', koaCors({
...
     } catch (err) {
         // Since there was an error somewhere down the middleware,
         // then we need to throw this client away.
-        pgConnection.done(err);
+        error = err;
 
         throw err;
+    } finally {
+        appLogger.log('debug', 'Closing DB connection');
+        pgConnection.done(error);
```
Comment (fzaninotto) [inline src/api/server.js]: `error` my not be defined here

Comment (Kmaschta) [inline src/api/server.js]: Where can I define it ?

Comment (fzaninotto) [inline src/api/server.js]: Ah OK it's in the closure, gotcha.

**Comment by djhi** [comment]
:+1:

**Pull request merged**